src/mmrelay/cli.py

```
@@ -328,8 +331,8 @@ def _validate_credentials_json(config_path: str) -> bool:
             return False
 
         return True
-    except Exception as e:
-        print(f"❌ Error: Could not validate credentials.json: {e}")
+    except (OSError, json.JSONDecodeError) as e:
+        logger.error("❌ Error: Could not validate credentials.json: %s", e)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)